SITCOMTN-076.tex

```diff
@@ -49,6 +53,325 @@
+
+\begin{enumerate}
+
+  \item the derived data product has been approved for release in one of the official Rubin Observatory outward-facing communication channels;
+
+  \item the derived data product is documented as part of an approved technote (see Section \ref{technotes}).
+
+\end{enumerate}
+
+Derived data products that represent visit, coadd, and difference images from LSSTCam on-sky commissioning, with the exception of alert postage stamp images, are embargoed for a period of at least 30 days following the observation.
+%not corresponding to artificial Earth-orbiting satellites
+
+%Derived data products based on commissioning on-sky data will not be released for at least 80 hours following the observation.
+
+During the on-sky commissioning period with LSSTCam, members of the Project team are allowed to discuss technical details of their work outside the team, and they may freely discuss aspects that do NOT relate to specific on-sky data products from LSSTCam or interim science performance.
+Discussion on the general status of commissioning should refer to Project-approved resources for information on the progress of commissioning activities (e.g., digests, news stories, published Project status on the Rubin Observatory website).
+
+\subsection{AuxTel, Electro-optical Datasets from ComCam and LSSTCam, and Non-proprietary Precursor Datasets}
+
+Derived data products resulting from analysis of AuxTel datasets, and electro-optical datasets from ComCam and LSSTCam, as well as non-proprietary precursor datasets (e.g., HSC and DECam) and non-proprietary simulated datasets (e.g., DESC DC2) may be openly discussed and shared.
+
+%\subsection{Transferring Data Products from Rubin Computing Facilities}
+
+%In general, it is NOT permitted to copy/move data products from Rubin Observatory computing facilities to other locations without explicit permission.
+%See Rubin Observatory Data Security Standards Implementation (DMTN-199).
+
+\subsection{Technotes}
+\label{technotes}
+
+\href{https://developer.lsst.io/project-docs/technotes.html}{Technotes} are a way for Rubin Observatory team members to write standalone documents that are native to the web, can be cited in literature, and are easy to write, publish, and update.
+A listing of technotes is available at \url{www.lsst.io}.
+
+During the on-sky commissioning period with LSSTCam, technotes are anticipated to be one of the primary mechanisms to share information about Rubin Observatory data with the science community.
+Project team members are encouraged to document their analyses in the form of technotes, e.g., to describe an algorithm or analysis software, to characterize and/or propose a solution to an outstanding issue, or to present the results of a science verification / validation study.
```

Comment:
See comment above about technical discussions.   These tech notes will certainly contain figures constructed from commissioning data -- that's the point.   What are the rules?

Reply:
Yes, the tech notes could include figures. As described in the section on Technotes:

> Prior to the associated Early Science release, technotes that involve proprietary on-sky ComCam and LSSTCam data must be first drafted in a restricted space and reviewed/approved by the Project using a standard checklist in a timely manner before posting. This process will ensure that released technotes meet basic standards for documentation quality and conform to Rubin Observatory information release policies. As living documents, technotes may describe work in progress and may be updated as studies progress. Once released, updated technote versions must maintain compliance with the approval checklist criteria.